### DISCUSSION TITLE
Public access problems

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#11784] [Help] [answered]

**Comment by JCV67** [comment]
How should the access permission to the users table, be set?
Thank you

**Reply by ConorWebb96** [reply, marked as answer]
Hey @JCV67,

Could you provide some screenshots of your screen role settings. Table settings etc. 
If its a public page the Users table isn't likely to be relevant unless you are using some user related information on the page.


To answer your other question you can invite users outside of your app. These can be assigned either admins, which will have access to everything or members whom will need to be assigned permissions on an app per app basis.

You can manage them within your apps by going into an app clicking on the users icon at the top right of your screen. This should open a side panel assign/un-assign users whom aren't admins with whatever role you wish.
<img width="233" alt="Screenshot 2023-09-15 at 13 23 37" src="https://github.com/Budibase/budibase/assets/126772285/7c0d543c-8ca1-400d-89ee-36290de1b97b">

You can create custom user roles on a app per app basis by going to the users table and clicking on the edit roles button.
<img width="1070" alt="Screenshot 2023-09-15 at 13 25 20" src="https://github.com/Budibase/budibase/assets/126772285/e7f5b396-78e2-43de-a7d9-f0dcb996905d">
This should open up a modal which will give you the option of create or update existing role's permissions.
<img width="401" alt="Screenshot 2023-09-15 at 13 25 43" src="https://github.com/Budibase/budibase/assets/126772285/e7fe7484-4b8c-4526-aff6-f59343b14076">


I hope this makes sense.

**Reply by JCV67** [reply]
Thank you for your reply @ConorWebb96. Just found out that one of the data sources was not set to public on write permissions, Before the lastest updates was working. I think after last changes on rbac the behavior changed a little. Now it is working.
Thank you
